### PR TITLE
Merge release v1.10.0-beta.1 into 'main'

### PR DESCRIPTION
**Pull request by ddimatos**
##### SUMMARY
Release tasks, addresses issue #1148

### CODE REVIEW

**Comment by ddimatos** [comment]
This has passed full regression, you can review the release artifacts folder we maintain to see that evidence.

File: plugins/action/zos_unarchive.py
Comment (fernandofloresg) [inline plugins/action/zos_unarchive.py:1]: need to update copyright year

File: plugins/filter/wtor.py
Comment (fernandofloresg) [inline plugins/filter/wtor.py:1]: need to update copyright year

File: plugins/module_utils/dd_statement.py
Comment (fernandofloresg) [inline plugins/module_utils/dd_statement.py:1]: update copyright year

File: plugins/module_utils/ickdsf.py
Comment (fernandofloresg) [inline plugins/module_utils/ickdsf.py:1]: copyright year

File: plugins/module_utils/import_handler.py
Comment (fernandofloresg) [inline plugins/module_utils/import_handler.py:1]: copyright year

File: plugins/module_utils/validation.py
Comment (fernandofloresg) [inline plugins/module_utils/validation.py:1]: copyright year

File: plugins/module_utils/zos_mvs_raw.py
Comment (fernandofloresg) [inline plugins/module_utils/zos_mvs_raw.py:1]: copyright year

File: plugins/modules/zos_job_query.py
Comment (fernandofloresg) [inline plugins/modules/zos_job_query.py:1]: update copyright years

**Review by fernandofloresg — CHANGES_REQUESTED** [review]

File: plugins/modules/zos_ping.rexx [hunk 1]
```diff
@@ -1,7 +1,7 @@
 /* rexx  __ANSIBLE_ENCODE_EBCDIC__  */
 /* WANT_JSON */
 
-/* Copyright (c) IBM Corporation 2019, 2020, 2023 */
+/* Copyright (c) IBM Corporation 2019, 2023 */
```
Comment (richp405) [inline plugins/modules/zos_ping.rexx:4]: 2024

File: plugins/action/zos_copy.py [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# Copyright (c) IBM Corporation 2019-2023
+# Copyright (c) IBM Corporation 2019, 2023
```
Comment (rexemin) [inline plugins/action/zos_copy.py:1]: Still need to update the copyright year to 2024 here

**Review by richp405 — CHANGES_REQUESTED** [review]
Minor -dates in copyrights.

**Review by rexemin — DISMISSED** [review]

**Comment by ddimatos** [comment]
Every one of the comments about the copyrights was corrected, prior to the PR review request, I am not sure what is happening, I am looking into it, below you can see the time stamp change, git status showing nothing is out sync and coyright is correct. 

image

File: plugins/action/zos_job_submit.py [hunk 3]
```diff
@@ -1,4 +1,4 @@
-# Copyright (c) IBM Corporation 2019, 2020, 2023
+# Copyright (c) IBM Corporation 2019, 2023
```
[the remark this answers is not in the document]
Comment (ddimatos) [reply plugins/action/zos_job_submit.py:1]: Done

File: plugins/action/zos_unarchive.py
Comment (ddimatos) [reply plugins/action/zos_unarchive.py:1]: Done

File: plugins/filter/wtor.py
Comment (ddimatos) [reply plugins/filter/wtor.py:1]: Done

File: plugins/module_utils/dd_statement.py
Comment (ddimatos) [reply plugins/module_utils/dd_statement.py:1]: Done

File: plugins/module_utils/ickdsf.py
Comment (ddimatos) [reply plugins/module_utils/ickdsf.py:1]: Done

File: plugins/module_utils/import_handler.py
Comment (ddimatos) [reply plugins/module_utils/import_handler.py:1]: Done

File: plugins/module_utils/validation.py
Comment (ddimatos) [reply plugins/module_utils/validation.py:1]: Done

File: plugins/module_utils/zos_mvs_raw.py
Comment (ddimatos) [reply plugins/module_utils/zos_mvs_raw.py:1]: Done

File: plugins/modules/zos_job_query.py
Comment (ddimatos) [reply plugins/modules/zos_job_query.py:1]: Done

File: tests/functional/modules/test_zos_data_set_func.py
[the remark this answers is not in the document]
Comment (ddimatos) [reply tests/functional/modules/test_zos_data_set_func.py:1]: Done

File: tests/functional/modules/test_zos_mvs_raw_func.py
[the remark this answers is not in the document]
Comment (ddimatos) [reply tests/functional/modules/test_zos_mvs_raw_func.py:1]: Done

File: tests/functional/modules/test_zos_job_output_func.py
[the remark this answers is not in the document]
Comment (ddimatos) [reply tests/functional/modules/test_zos_job_output_func.py:1]: Done

File: docs/source/requirements-single.rst [hunk 4]
```diff
@@ -32,6 +32,7 @@ The managed z/OS node is the host that is managed by Ansible, as identified in
 the Ansible inventory. For the **IBM z/OS core collection** to manage the z/OS node,
 some dependencies are required to be installed on z/OS such as:
 
+* `z/OS`_
```
[the remark this answers is not in the document]
Comment (ddimatos) [reply docs/source/requirements-single.rst:35]: Done

File: plugins/action/zos_copy.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (ddimatos) [reply plugins/action/zos_copy.py:1]: Done

File: plugins/action/zos_job_submit.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (ddimatos) [reply plugins/action/zos_job_submit.py:1]: Done

File: plugins/modules/zos_job_query.py [hunk 5]
```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/python
 # -*- coding: utf-8 -*-
 
-# Copyright (c) IBM Corporation 2019, 2020, 2023
+# Copyright (c) IBM Corporation 2019, 2023
```
[the remark this answers is not in the document]
Comment (ddimatos) [reply plugins/modules/zos_job_query.py:4]: Done

File: plugins/modules/zos_ping.rexx (hunk 1 above)
Comment (ddimatos) [reply plugins/modules/zos_ping.rexx:4]: Done but no , it should be 2023 , the change was to move to the proper year structure for copy rights, where first year is when it was created, followed by a comma and then the last year it was edited; I edited in 2023 but never removed the middle year 2020.

File: plugins/action/zos_copy.py (hunk 2 above)
Comment (ddimatos) [reply plugins/action/zos_copy.py:1]: Done

**Review by fernandofloresg — APPROVED** [review]

File: plugins/action/zos_job_submit.py (hunk 3 above)
[the remark this answers is not in the document]
Comment (richp405) [reply plugins/action/zos_job_submit.py]: still needs 2024

**Review by richp405 — APPROVED** [review]

File: plugins/modules/zos_ping.rexx (hunk 1 above)
Comment (richp405) [reply plugins/modules/zos_ping.rexx:4]: got it

**Review by richp405 — APPROVED** [review]

**Pull request merged**